src/Compendium/Compendium.jsx

```diff
@@ -0,0 +1,89 @@
+import {  useState, useEffect } from "react";
```

Comment:
try to keep your import block organized by type: libraries, css, utils, components